Gemfile

```diff
@@ -8,7 +8,8 @@ gem 'rouge'
 gem 'go_script'
 
 group :jekyll_plugins do
-  gem 'guides_style_18f', '~> 1.0.4'
```

Comment:
I wonder if it's a version problem. Can we still use 1.0.x? is there a newer version?

Reply:
This is a removed line.  I removed the version spec to see how high it'd go.  2.0.1, I'll link it below in the Gemfile.lock

Reply:
that's what I mean... right now it's open. What happens if we stick with 1.x?

Reply:
using the following should work:
gem "guides_style_18f", tag: 'v1.0.6

Reply:
Needs to be `tag: 'refs/tags/v1.0.6` because you have a branch of the same name :).
No difference though, I see the same errors running Jekyll.  Jekyll did get updated too, but it's being flagged by Github Security alerts.  I'll try to dig into the actual error a bit